### PR TITLE
 {bio}(foss/2015b) BioKanga 4.3.4, 4.3.5 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**
- Added EasyConfig for BioKanga 4.3.4; updating location of source and building

### CODE REVIEW

File: easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['https://github.com/csiro-crop-informatics/biokanga/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = ['BioKanga-4.3.4_configure.patch']
+
+builddependencies = [
+    ('Autoconf', '2.69'),
+    ('Automake', '1.15')
```
Comment (boegel) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb]: @nathanhaigh Do you mind changing this to `Autotools` `20150215`? You can either use the existing `Autotools-20150215-GCC-4.9.3-2.25.eb` easyconfig for that (which EasyBuild v3.x will pick up automatically when using `foss/2015b`), or add the additional necessary easyconfigs for `Autotools-20150215-foss-2015b.eb`

File: easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+description = """BioKanga is an integrated toolkit of high performance bioinformatics
+  subprocesses targeting the challenges of next generation sequencing analytics.
+  Kanga is an acronym standing for 'K-mer Adaptive Next Generation Aligner'."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['https://github.com/csiro-crop-informatics/biokanga/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = ['BioKanga-4.3.4_configure.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb]: please avoid hardcoding the version, use `BioKanga-%(version)s_configure.patch`

File: easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4_configure.patch [hunk 3]
```diff
@@ -0,0 +1,29 @@
+--- ./configure.ac	2017-04-04 11:47:18.067209892 +0930
```
Comment (boegel) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4_configure.patch]: @nathanhaigh please add a short description at the top of the patch file, and mention who created it (or include a reference to upstream fix)

File: easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb (hunk 2 above)
Comment (nathanhaigh) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb]: done

File: easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb (hunk 1 above)
Comment (nathanhaigh) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4-foss-2015b.eb]: done

File: easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4_configure.patch (hunk 3 above)
Comment (nathanhaigh) [inline easybuild/easyconfigs/b/BioKanga/BioKanga-4.3.4_configure.patch]: done

**Comment by nathanhaigh** [comment]
@boegel ping

**Comment by boegel** [comment]
lgtm, test reports coming up...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2478.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7fc1f47fb39ba6fdcecf82b7183b0d28 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2022.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f472222eb15656bf3c947c0d5f5da82c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @nathanhaigh!

**Pull request merged**